docs/application/tizen-studio/TOC.md

```diff
@@ -130,7 +131,7 @@
 # [Configurable SDK](configurable-sdk/configurable-sdk.md)
 
 # Release Notes
```

Comment:
Don't we need to add Release note for 2.2 here? I'm asking as the tizen-studio/index.md page is updated and including  information of 2.2.